Jenkinsfile

```diff
@@ -96,6 +96,12 @@ pipeline {
             choices: ['--info', '--debug', '']
         )
 
+        booleanParam(
```

Comment:
Commit message: "optional", "to see exceptions"

Reply:
Done.